portfolio/src/main/webapp/index.html

```diff
@@ -145,7 +145,20 @@ <h2> Comments </h2>
 
                     <input type="submit" />
                 </form>
+                <form action="/updateComments" method="POST">
```

Comment:
Following on that, IMO, we can store the max number in datastore as well so that we don't need to make them in the same java file i.e. same doPost() function